modules/virt-setting-node-maintenance-cli.adoc

```diff
@@ -5,34 +5,21 @@
 [id="virt-setting-node-maintenance-cli_{context}"]
 = Setting a node to maintenance mode in the CLI
 
-Set a node to maintenance mode by creating a `NodeMaintenance` custom resource
-(CR) object that references the node name and the reason for setting it to
-maintenance mode.
+Set a node to maintenance mode by marking it as unschedulable and using the `oc adm drain` command to evict or delete pods from the node.
 
 .Procedure
 
-. Create the node maintenance CR configuration. This example uses a CR that is
-called `node02-maintenance.yaml`:
+. Mark the node as unschedulable. The node status will change to `NotReady,SchedulingDisabled`.
 +
-[source,yaml]
+[source,terminal]
 ----
-apiVersion: nodemaintenance.kubevirt.io/v1beta1
-kind: NodeMaintenance
-metadata:
-  name: node02-maintenance
-spec:
-  nodeName: node02
-  reason: "Replacing node02"
+$ oc adm cordon <node1>
 ----
 
-. Create the `NodeMaintenance` object in the cluster:
+. Drain the node in preparation for maintenance. The node live migrates virtual machine instances that have the `evictionStrategy` field set to `LiveMigrate`. All other pods and virtual machines on the node are deleted and
```

Comment:
Having 'evictionStrategy: LiveMigrate` is not enough for a VM to be live migrated. The VM Instance must have the 'LiveMigratable' condition with status set to 'True'